src/main/java/uk/gov/digital/ho/hocs/info/AppContextEventListener.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.digital.ho.hocs.info;
```

Comment:
We need to remove this, we should not print properties to logs. I think this was added temporary to guide with environment issues.